test/csv/csv_preserve_trailing_spaces.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Get existing CSV file with trailing spaces in `string,array`",
+    "request": {
+        "server_url": "{{ datastore "req_base_url" }}",
+        "endpoint": "bounce-json",
+        "method": "POST",
+        "body": {{ file_csv "../_res/assets/trailing_spaces_in_string_array.csv" ',' | marshal }}
+    },
```

Comment:
all files for the test should reside in the test directory, if not shared with other tests.